low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -63,9 +63,23 @@ class SSPLDevDeploy:
         """Initialize required config variables."""
         self.args = args
         self.CORTX_MONITOR_BASE_URL = "https://raw.githubusercontent.com/Seagate/cortx-monitor/main"
-        self.build_url = ("http://cortx-storage.colo.seagate.com/releases"
+        file = "/etc/system-release"
+        with open(file) as fObj:
+            content = fObj.read()
+        if "CentOS Linux release 7.9" in content:
+            self.build_url = ("http://cortx-storage.colo.seagate.com/releases"
+                          "/cortx/github/main/centos-7.9.2009"
+                          "/last_successful_prod/")
+            self.cortx_utils_build_url = ("http://cortx-storage.colo.seagate.com/releases/"
+                     "cortx/components/github/main/centos-7.9.2009/"
+                     "dev/cortx-utils/last_successful/")
+        else:
+            self.build_url = ("http://cortx-storage.colo.seagate.com/releases"
                           "/cortx/github/main/centos-7.8.2003"
                           "/last_successful_prod/")
+            self.cortx_utils_build_url = ("http://cortx-storage.colo.seagate.com/releases/"
+                     "cortx/components/github/main/centos-7.8.2003/"
+                     "dev/cortx-utils/last_successful/")
```

Comment:
Instead of repeating the code for each OS, I think we should have a configurable path. Something like:
```
os_name = "centos-7.8.2003" # need to get this dynamically
self.build_url = f"http://cortx-storage.colo.seagate.com/releases/cortx/github/main/{os_name}/last_successful_prod/"
```
Not sure if this is possible without much changes since we are reading OS from `/etc/system-release`
Please check, if not then I am okay.

Reply:
Sure. will take care as part cleanup work once we migrate to 7.9

Reply:
Okay